code/datums/martial/_martial.dm

```diff
@@ -19,6 +19,9 @@
 /datum/martial_art/proc/disarm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
 	return FALSE
 
+/datum/martial_art/proc/help_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	return FALSE
```

Comment:
not modular. Needs either to be modular or have //skyrat edit

Reply:
will do